codemods/babel-plugin-codemod-object-assign-to-object-spread/src/index.js

```diff
@@ -2,7 +2,7 @@ import syntaxObjectRestSpread from "@babel/plugin-syntax-object-rest-spread";
 
 export default function ({ types: t }) {
   return {
-    inherits: syntaxObjectRestSpread,
+    inherits: syntaxObjectRestSpread.default,
```

Comment:
These syntax plugins are from packages outside of the monorepo which are and will always be CommonJS. With this PR we fully match the Node.js behavior, so we need to add `.default` in order to get `module.exports`.